packages/@lwc/engine-server/src/index.ts

```diff
@@ -10,7 +10,10 @@ import './polyfills';
 // Renderer initialization -------------------------------------------------------------------------
 import './initializeRenderer';
 
+import { parseFragment } from '@lwc/engine-core';
+
 // Engine-core public APIs -------------------------------------------------------------------------
+export { parseFragment as parseSVGFragment };
```

Comment:
why do we reuse the same parseFragment in server?

Reply:
It looks like the reason is because `parseSVGFragment` uses `getFirstChild`, which is not supported in the server environment:

https://github.com/salesforce/lwc/blob/4d5b57ee33ad3254cb7bd9d5e3393a8d0a23f781/packages/%40lwc/engine-core/src/framework/template.ts#L182-L186

If I try to use the actual `parseSVGFragment`, I get an error in the Jest tests:

<details><summary>Click to see</summary>

```
Summary of all failing tests
 FAIL  packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts (18.993 s)
  ● fixtures › svgs/index.js

    TypeError: "getFirstChild" is not supported in this environment

      13 |     create,
      14 |     StringToLowerCase,
    > 15 |     htmlPropertyToAttribute,
         |               ^
      16 |     noop,
      17 | } from '@lwc/shared';
      18 |

      at src/renderer.ts:15:15
      at ../engine-core/dist/engine-core.cjs.js:4654:12
      at ../engine-core/dist/engine-core.cjs.js:4645:31
      at tmpl (src/__tests__/fixtures/svgs/dist/compiled.js:35:27)
      at ../engine-core/dist/engine-core.cjs.js:4718:27
      at ReactiveObserver.observe (../engine-core/dist/engine-core.cjs.js:339:13)
      at isUpdatingTemplate (../engine-core/dist/engine-core.cjs.js:4674:13)
      at runWithBoundaryProtection (../engine-core/dist/engine-core.cjs.js:5534:5)
      at evaluateTemplate (../engine-core/dist/engine-core.cjs.js:4667:5)
      at invokeComponentRenderMethod (../engine-core/dist/engine-core.cjs.js:4820:41)
```

</details>